clients/drcachesim/tools/opcode_mix.h

```diff
@@ -44,6 +44,7 @@
 
 #include "dr_api.h" // Must be before trace_entry.h from analysis_tool.h.
 #include "analysis_tool.h"
+#include "decode_cache.h"
```

Comment:
Please add a changelog entry in release.dox for this new feature.

Reply:
Done